Packs/FreeEnrichers/pack_metadata.json

```diff
@@ -66,21 +66,21 @@
             "mandatory": false,
             "display_name": "Data Mining for Threat Intelligence"
         },
-        "URLhaus": {
+        "URLHaus": {
             "mandatory": false,
-            "display_name": "URLhaus has the goal of sharing malicious URLs that are being used for malware distribution."
+            "display_name": "URLHaus has the goal of sharing malicious URLs that are being used for malware distribution."
         },
         "Whois": {
             "mandatory": false,
             "display_name": "This Content Pack helps you run Whois commands as playbook tasks or real-time actions within Cortex XSOAR to obtain valuable domain metadata."
         },
 
-        "AbuseIPDB": {
+        "AbuseDB": {
             "mandatory": false,
             "display_name": "Central repository to report and identify IP addresses that have been associated with malicious activity online. Check the Detailed Information section for more information on how to configure the integration."
         },
 
-        "AlienVault OTX": {
+        "AlienVault_OTX_v2": {
```

Comment:
it should be the pack name, not the name of the integrations pack

Reply:
It's the pack folder name, I copied the wrong one in that one.